src/Projection/ProjectorStore/DatabaseStore.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Patchlevel\EventSourcing\Projection\ProjectorStore;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\DBAL\Types\Types;
+use Patchlevel\EventSourcing\Projection\ProjectorId;
+use Patchlevel\EventSourcing\Projection\ProjectorStatus;
+use Patchlevel\EventSourcing\Schema\SchemaConfigurator;
+
+use function array_map;
+
+final class DatabaseStore implements ProjectorStore, SchemaConfigurator
```

Comment:
Maybe change the naming to `DBALStore` or `DoctrineStore`? Since it's more that then a general `DatabaseStore`.